source/plot/Axis.ooc

```diff
@@ -42,56 +42,63 @@ Axis: class {
 		result
 	}
 
-	getSVG: func (plotAreaSize, axisAreaSize, position, translationToOrigo, scaling: FloatPoint2D) -> String {
+	getSVG: func (plotAreaSize, margin, translationToOrigo, scaling: FloatPoint2D, fontSize: Int) -> String {
```

Comment:
By convention, acronyms with more than 3 letters only capitalize the first letter, so `getSvg`.